src/pgduckdb_duckdb.cpp

```diff
@@ -298,19 +312,19 @@ DuckDBManager::LoadExtensions(duckdb::ClientContext &context) {
 
 void
 DuckDBManager::RefreshConnectionState(duckdb::ClientContext &context) {
+	const auto extensions_table_last_seq = GetSeqLastValue("extensions_table_seq");
+	if (IsExtensionsSeqLessThan(extensions_table_last_seq)) {
+		LoadExtensions(context);
+		UpdateExtensionsSeq(extensions_table_last_seq);
+	}
```

Comment:
Why move this around? (it seems fine, but I'm curious what the reason is)

Reply:
Because we need Azure extension to be loaded to be able to `CREATE SECRET(TYPE=Azure,...)` :)